runtime/util/vmargs.c

```diff
@@ -506,10 +508,16 @@ optionValueOperations(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, IDATA
 				done = TRUE;
 				break;
 
+			case GET_DBL_VALUE:
```

Comment:
Adding GET_DBL_VALUE option here makes code look ugly. Would you please try to implement independent case for GET_DBL_VALUE. Duplicated code like scanStart setup can be moved to separate function(s) and be called in both places. I believe it would be easier to read.